server/routes/private.js

```diff
@@ -49,6 +50,41 @@ router.post('/api/book-appointment', async (req, res) => {
 
 });
 
+router.post('/api/recurring_schedule', async (req, res) => {
+    // Validate appropriate parameters are passed into the book appointment endpoint.
+    // Verification to make sure appointments are only made for future dates and that the worker is available during that time will be handled by the front end.
+    const paramSchema = Joi.object({
+        
```

Comment:
Please remove unnecessary whitespace to increase code readability for the future.

Reply:
Thanks @catai9 I've fixed the unnecessary whitespaces!